api/src/pcapi/alembic/versions/20240116T115259_36cf4c5380ad_add_partial_index_for_musictype_in_offer_extradata.py

```diff
@@ -0,0 +1,40 @@
+"""
+Add partial index for musicType in Offer.extraData
+"""
+from alembic import op
+
+from pcapi import settings
+
+
+# pre/post deployment: post
+# revision identifiers, used by Alembic.
+revision = "36cf4c5380ad"
+down_revision = "3467ee234047"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.execute("COMMIT")
+    op.execute("""SET SESSION statement_timeout = '2400s'""")
+    op.execute(
+        """
+            CREATE INDEX CONCURRENTLY IF NOT EXISTS
+            offer_music_type_idx ON public.offer USING btree (("jsonData" ->> 'musicType'::text))
+            WHERE (("jsonData" ->> 'musicType'::text)) IS NOT NULL;
+        """
+    )
+    op.execute(
+        f"""
+                SET SESSION statement_timeout={settings.DATABASE_STATEMENT_TIMEOUT}
+            """
+    )
```

Comment:
Pourquoi cette indentation ? Comme je te le disais, une PR doit être mise au propre avant d'être soumise.
Ici et encore plus dans les 3 autres migrations.
```suggestion
    op.execute(f"SET SESSION statement_timeout={settings.DATABASE_STATEMENT_TIMEOUT}")
```